File: testcases/paper/NestedAlias/NestedAlias.py
class NestedAlias:
    def original_method(self):
        self.i = 1
        print("Original method")

    # Create an alias for the original_method
    alias_method = original_method

    def call_alias(self):
        self.j = 2
        self.alias_method()
        pass

File: testcases/paper/NestedAlias/NestedAlias.go
package NestedAlias

type NestedAlias struct {
    i,j int
}

func (c NestedAlias) method() {
    c.i = 1
}

func (c NestedAlias) method2() {
    c.j = 2
    var aliasMethod = c.method
    aliasMethod()
    return
}